### ISSUE TITLE
Storage snapshot: optional skipHead parameter not optional

### ISSUE THREAD

**Issue by beorn7** [prometheus/prometheus#4732]
## Bug Report
**What did you do?**

```
$ curl -XPOST http://localhost:9090/api/v1/admin/tsdb/snapshot
```

**What did you expect to see?**

```
{"status":"success","data":{"name":"20181012T143717Z-655852ac65b7132f"}}
```

**What did you see instead? Under which circumstances?**

```
{"status":"error","errorType":"unavailable","error":"unable to parse boolean 'skip_head' argument: strconv.ParseBool: parsing \"\": invalid syntax"}
```

**Work around**

Explicitly specify the "optional" parameter:

```
$ curl -XPOST http://localhost:9090/api/v1/admin/tsdb/snapshot?skip_head=false
```

**Environment**

* Prometheus version:

```
prometheus, version 2.4.2 (branch: HEAD, revision: c305ffaa092e94e9d2dbbddf8226c4813b1190a0)
  build user:       root@dcde2b74c858
  build date:       20180921-07:22:29
  go version:       go1.10.3
```

**Comment by brian-brazil** [comment]
Dupe of #4671, this is fixed in 2.4.3.

**Comment by brian-brazil** [comment]
Don't worry about it, it happens to us all.
